### ISSUE TITLE
Create-react-app failed to minify 

### ISSUE THREAD

**Issue by saeedalzahrani** [femioladeji/react-slideshow#39]
Error message
>Failed to minify the code from this file: ./node_modules/react-slideshow-image/lib/helpers.js:10 

File
```js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.getUnhandledProps = getUnhandledProps;

function getUnhandledProps(ComponentProps, props) {
  const handledProps = Object.keys(ComponentProps);
  return Object.keys(props).reduce((acc, prop) => {
    if (handledProps.indexOf(prop) === -1) acc[prop] = props[prop];
    return acc;
  }, {});
}
```
Version
```
"react-slideshow-image": "^1.2.0",
```

package.json
```js
{
  "name": "react-auth-template",
  "homepage": "name",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "autoprefixer": "7.1.6",
    "axios": "^0.18.0",
    "bootstrap": "^4.3.1",
    "chalk": "1.1.3",
    "dotenv": "4.0.0",
    "dotenv-expand": "4.2.0",
    "each-async": "^1.1.1",
    "fs-extra": "3.0.1",
    "gh-pages": "^2.0.1",
    "object-assign": "4.1.1",
    "promise": "8.0.1",
    "raf": "3.4.0",
    "react": "^16.4.2",
    "react-background-slider": "^1.2.0",
    "react-bootstrap": "^1.0.0-beta.5",
    "react-dom": "^16.4.2",
    "react-router-dom": "^4.3.1",
    "react-slideshow-image": "^1.2.0",
    "resolve": "1.6.0",
    "url-loader": "0.6.2",
    "react-scripts": "2.0.0-next.3e165448"
  },
  "scripts": {
    "start": "node scripts/start.js",
    "build": "node scripts/build.js",
    "test": "node scripts/test.js --env=jsdom",
    "make-standard": "eslint --fix ./src;exit 0",
    "nag": "eslint ./src;exit 0",
    "deploy": "gh-pages -d build",
    "predeploy": "npm run build"
  },
  "jest": {
    "collectCoverageFrom": [
      "src/**/*.{js,jsx,mjs}"
    ],
    "setupFiles": [
      "<rootDir>/config/polyfills.js"
    ],
    "testMatch": [
      "<rootDir>/src/**/__tests__/**/*.{js,jsx,mjs}",
      "<rootDir>/src/**/?(*.)(spec|test).{js,jsx,mjs}"
    ],
    "testEnvironment": "node",
    "testURL": "http://localhost",
    "transform": {
      "^.+\\.(js|jsx|mjs)$": "<rootDir>/node_modules/babel-jest",
      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
      "^(?!.*\\.(js|jsx|mjs|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
    },
    "transformIgnorePatterns": [
      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs)$"
    ],
    "moduleNameMapper": {
      "^react-native$": "react-native-web"
    },
    "moduleFileExtensions": [
      "web.js",
      "js",
      "json",
      "web.jsx",
      "jsx",
      "node",
      "mjs"
    ]
  },
  "babel": {
    "presets": [
      "react-app"
    ]
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "devDependencies": {
    "ajv": "^5.5.2",
    "babel-core": "6.26.0",
    "babel-eslint": "^9.0.0",
    "babel-jest": "20.0.3",
    "babel-loader": "7.1.2",
    "babel-preset-react-app": "^3.1.1",
    "babel-runtime": "6.26.0",
    "case-sensitive-paths-webpack-plugin": "2.1.1",
    "css-loader": "0.28.7",
    "eslint": "^5.14.1",
    "eslint-config-react-app": "^3.0.7",
    "eslint-config-standard": "^12.0.0",
    "eslint-loader": "^2.1.2",
    "eslint-plugin-flowtype": "^2.50.3",
    "eslint-plugin-import": "^2.13.0",
    "eslint-plugin-jsx-a11y": "^6.2.1",
    "eslint-plugin-node": "^8.0.1",
    "eslint-plugin-promise": "^4.0.1",
    "eslint-plugin-react": "^7.12.0",
    "eslint-plugin-standard": "^4.0.0",
    "extract-text-webpack-plugin": "3.0.2",
    "file-loader": "1.1.5",
    "html-webpack-plugin": "2.29.0",
    "jest": "20.0.4",
    "node-sass": "^4.9.3",
    "postcss-flexbugs-fixes": "3.2.0",
    "postcss-loader": "2.0.8",
    "react-dev-utils": "^5.0.1",
    "sass-loader": "^7.1.0",
    "style-loader": "0.19.0",
    "sw-precache-webpack-plugin": "0.11.4",
    "webpack": "3.8.1",
    "webpack-dev-server": "2.9.4",
    "webpack-manifest-plugin": "1.3.2"
  }
}
```

**Comment by femioladeji** [comment]
@saeedalzahrani thanks for raising this issue. I'll take a look at it. 👍

**Comment by femioladeji** [comment]
@saeedalzahrani I'm guessing you did `npm run eject`. Did you make some changes to your configuration and what command do you run that generates this error? I'm trying to reproduce the bug but I couldn't

**Comment by saeedalzahrani** [comment]
I am using react-gh-pages to deploy. https://github.com/gitname/react-gh-pages

Here is the full error message
```
Saeeds-MacBook-Pro:react-template saeedalzahrani$ npm run deploy

> react-auth-template@0.1.0 predeploy /Users/saeedalzahrani/Desktop/wdi/projects/final_project/react-template
> npm run build


> react-auth-template@0.1.0 build /Users/saeedalzahrani/Desktop/wdi/projects/final_project/react-template
> node scripts/build.js

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/react-slideshow-image/lib/helpers.js:10 

Read more here: http://bit.ly/2tRViJ9

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-auth-template@0.1.0 build: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-auth-template@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/saeedalzahrani/.npm/_logs/2019-07-31T14_28_08_479Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-auth-template@0.1.0 predeploy: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-auth-template@0.1.0 predeploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/saeedalzahrani/.npm/_logs/2019-07-31T14_28_08_560Z-debug.log
```

And the debug log

```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/local/Cellar/node/12.4.0/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'deploy'
1 verbose cli ]
2 info using npm@6.9.0
3 info using node@v12.4.0
4 verbose run-script [ 'predeploy', 'deploy', 'postdeploy' ]
5 info lifecycle react-auth-template@0.1.0~predeploy: react-auth-template@0.1.0
6 verbose lifecycle react-auth-template@0.1.0~predeploy: unsafe-perm in lifecycle true
7 verbose lifecycle react-auth-template@0.1.0~predeploy: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/saeedalzahrani/Desktop/wdi/projects/final_project/react-template/node_modules/.bin:/Users/saeedalzahrani/.rbenv/shims:/Library/Frameworks/Python.framework/Versions/3.7/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
8 verbose lifecycle react-auth-template@0.1.0~predeploy: CWD: /Users/saeedalzahrani/Desktop/wdi/projects/final_project/react-template
9 silly lifecycle react-auth-template@0.1.0~predeploy: Args: [ '-c', 'npm run build' ]
10 silly lifecycle react-auth-template@0.1.0~predeploy: Returned: code: 1  signal: null
11 info lifecycle react-auth-template@0.1.0~predeploy: Failed to exec predeploy script
12 verbose stack Error: react-auth-template@0.1.0 predeploy: `npm run build`
12 verbose stack Exit status 1
12 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
12 verbose stack     at EventEmitter.emit (events.js:200:13)
12 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
12 verbose stack     at ChildProcess.emit (events.js:200:13)
12 verbose stack     at maybeClose (internal/child_process.js:1021:16)
12 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
13 verbose pkgid react-auth-template@0.1.0
14 verbose cwd /Users/saeedalzahrani/Desktop/wdi/projects/final_project/react-template
15 verbose Darwin 18.7.0
16 verbose argv "/usr/local/Cellar/node/12.4.0/bin/node" "/usr/local/bin/npm" "run" "deploy"
17 verbose node v12.4.0
18 verbose npm  v6.9.0
19 error code ELIFECYCLE
20 error errno 1
21 error react-auth-template@0.1.0 predeploy: `npm run build`
21 error Exit status 1
22 error Failed at the react-auth-template@0.1.0 predeploy script.
22 error This is probably not a problem with npm. There is likely additional logging output above.
23 verbose exit [ 1, true ]

```

This error also occurs when I run `npm run build` directly.

This is my repository if it is helpful to clone and reproduce https://github.com/salt-1/project-4-frontend

**Comment by femioladeji** [comment]
@saeedalzahrani the issue has been fixed now in this PR #40 and a new version has been released. Use the v1.2.1 in your package.json and you should be good to go. Thanks for the issue
